.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Libraries
 /vendor/*
+!/vendor/.htaccess
```

Comment:
Isn't this covered by https://github.com/forkcms/forkcms/blob/master/.htaccess#L17